securedrop-workstation-svs-disp/debian/changelog-buster

```diff
@@ -1,3 +1,9 @@
+securedrop-workstation-svs-disp (0.2.2+buster) unstable; urgency=medium
+
+  * Fix mime type handling for desktop files
+
+ -- SecureDrop Team <user@localhost>  Fri, 20 Mar 2020 10:43:41 -0400
```

Comment:
> SecureDrop Team <user@localhost>

should be:

> SecureDrop Team <securedrop@freedom.press> 

same as in the `securedrop-export` changelog. Will fix as part of review. Looks like the update-changelog script doesn't properly default to these values, will patch that as well.